docs/site/content/docs/latest/tanzu-debugging-tips.md

```diff
@@ -101,3 +95,24 @@ management cluster.
     docker stop 2f505b8b0c8a && docker rm 2f505b8b0c8a
     docker stop 3dd82b26ac04 && docker rm 3dd82b26ac04
     ```
+## Aditional helpful commands for debugging clusters
+1. Get all Cluster API related objects:
+```sh
```

Comment:
Same between all of these. Will need to also indent the codeblocks so they line up with the first letter of the text of the bullet (so add three leading spaces to account for the `1. `).

Reply:
done